content/en/docs/v3.3/op-guide/runtime-configuration.md

```diff
@@ -6,7 +6,7 @@ etcd comes with support for incremental runtime reconfiguration, which allows us
 
 Reconfiguration requests can only be processed when a majority of cluster members are functioning. It is **highly recommended** to always have a cluster size greater than two in production. It is unsafe to remove a member from a two member cluster. The majority of a two member cluster is also two. If there is a failure during the removal process, the cluster might not be able to make progress and need to [restart from majority failure][majority failure].
 
-To better understand the design behind runtime reconfiguration, please read [the runtime reconfiguration document][runtime-reconf].
+To better understand the design behind runtime reconfiguration, please read [the runtime reconfiguration document][../../runtime-reconf].
```

Comment:
Ditto, missed this the first time around: this link shouldn't have been changed. Revert the change to this line, and change the definition of `[runtime-reconf]` on line 175 below.